go-controller/pkg/util/node_annotations.go

```diff
@@ -51,6 +51,12 @@ const (
 
 	// ovnNodeChassisID is the systemID of the node needed for creating L3 gateway
 	ovnNodeChassisID = "k8s.ovn.org/node-chassis-id"
+
+	// ovnNodeCIDR is the CIDR form representation of primary network interface's attached IP address (i.e: 192.168.126.31/24 or 0:0:0:0:0:feff:c0a8:8e0c/64)
+	ovnNodeCIDR = "k8s.ovn.org/node-primary-cidr"
```

Comment:
say "IFAddr"/"ifaddr", not "cidr"; We are trying to fix things to only use "CIDR" to refer to _subnets_, and use "ifaddr" ("interface address") to refer to IP addresses in CIDR notation. (This applies to a few places in a few commits)